index.html

```diff
@@ -0,0 +1,228 @@
+              </div>
+          </div>
+        </div>
+      </div>
+    </head>
+    <main>
+      <div class="words-presentation" id="main-program">
+        <div class="main-program">
+          <p>Main Program</p>
+        </div>
+
+        <div class="border"></div>
+      
+        <div class="insight-section">
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_01.png" alt="lecture-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  Lecture
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Listen to the speakers from various countries about the messages of sharing and opening.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_02.png" alt="exhibition-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  DMT Exhibition
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Appreciate various creations applying DMT, organized from football Center Akwa Ibom.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_03.png" alt="forum-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  Forum
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Have the time to share your thoughts and opinions with expert for each topic.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_04.png" alt="workshop-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  Workshop
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Try creating your own work using open source license rather than just watching.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="d-flex d-flex-3 items">
+            <img src="./images/program_icon_05.png" alt="ignite-icon">
+            <div class="d-flex-3">
+              <div>
+                <p class="span-1">
+                  DMT Ignite
+                </p>
+              </div>
+              <div>
+                <p class="paragraph">
+                  Get opportunities to network with DMT affiliate around the world, also after the summit.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
```

Comment:
- [ ] Kindly align the contents in the  _**Main Program**_ section. Aligning the contents will make your design look identical to that of the original design and it will improve the visuals of the project. 👍 💻 

### Your Design (Mobile View)

![image](https://user-images.githubusercontent.com/77106962/218111536-637f28db-31f9-4f5b-8fff-3b75f98bf5bd.png)

### Behance Design (Mobile View)

![image](https://user-images.githubusercontent.com/77106962/218111676-11f61fc3-ee1e-4f80-99a8-72a2870c0dac.png)

### Your Design (Desktop View)

![image](https://user-images.githubusercontent.com/77106962/218111942-4ab57f0e-fd25-410a-a891-a13a2b6a88ca.png)

### Behance Design (Desktop View)

![image](https://user-images.githubusercontent.com/77106962/218112100-3f0ad4dc-75bf-439e-8173-fc3fef8f88eb.png)

#### Hint: Pay more attention to the _**description/paragraph**_ of each card. It would be nice to align them. You can also use the CSS grid layout to align items. 👍 💻

Reply:
Done